examples/systemd/rest-server.service

```diff
@@ -57,6 +57,8 @@ RestrictRealtime=true
 # if your service crashes with "code=killed, status=31/SYS", you probably tried to run linux_i386 (32bit) binary on a amd64 host
 SystemCallArchitectures=native
 SystemCallFilter=@system-service
+SystemCallFilter=~@resources
```

Comment:
For me `~@resources` leads to a crash on startup due to https://github.com/golang/go/issues/46279 .